src/main/java/org/eclipse/tractusx/ssi/lib/jwt/SignedJwtVerifier.java

```diff
@@ -61,16 +60,15 @@ public class SignedJwtVerifier {
    *
    * @param jwt a {@link SignedJWT} that was sent by the claiming party.
    * @return true if verified, false otherwise
-   * @throws DidParseException
-   * @throws SignatureException
-   * @throws DidResolverException
-   * @throws SignatureVerificationException
-   * @throws UnsupportedVerificationMethodException
-   * @throws SignatureParseException
+   * @throws DidParseException the did parse error
+   * @throws DidResolverException the did resolve error
+   * @throws SignatureVerificationException the signature verification error
+   * @throws UnsupportedVerificationMethodException the unsupported verification error
```

Comment:
This exception has been removed from the signature, but is still in the doc string.